includes/utilities.inc

```diff
@@ -116,9 +116,9 @@ EOQ;
 function islandora_paged_content_get_pages_solr(AbstractObject $object) {
   $pages = array();
   $qp = new islandoraSolrQueryProcessor();
-  $qp->buildQuery(format_string('@field:"@pid"', array(
+  $qp->buildQuery(format_string('@field:("@pid" OR "info:fedora/@pid")', array(
     '@field' => variable_get('islandora_solr_member_of_field', 'RELS_EXT_isMemberOf_uri_ms'),
-    '@pid' => "info:fedora/{$object->id}",
+    '@pid' => $object->id,
```

Comment:
Know this is what's here but these should probably be `!`s to pass through as opposed to `@`s. Other than code is sound.